File: src/main/java/org/pl/parser/ast/BoolNode.java
package org.pl.parser.ast;


/**
 * Single boolean value.
 */
public class BoolNode implements INode {
    public final boolean value;

    public BoolNode(boolean value) {
        this.value = value;
    }

    @Override
    public String toString() {
        return "BoolNode{val=" + value + '}';
    }
}

File: src/main/kotlin/org/leaflang/parser/ast/value/BoolNode.kt
package org.leaflang.parser.ast.value

import org.leaflang.parser.ast.INode
import org.leaflang.parser.utils.NodePosition

/**
 * Single boolean value.
 */
class BoolNode(override val position: NodePosition,
               val value: Boolean) : INode {
    override fun toString() = "BoolNode{val=$value}"
}